### PR TITLE
Feature/windows

### PR DESCRIPTION
**Pull request by pawelfor**
Make Windows work again

### CODE REVIEW

File: industry2/__main__.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
+import asyncio
+import sys
+
 from PyQt5.QtWidgets import QApplication
 
 from industry2.factory_gui import MainWindow
 
+
 if __name__ == '__main__':
```
Comment (Sorebit) [inline industry2/__main__.py:9]: Comment pls?

**Review by Sorebit — COMMENTED** [review]
Comment pls

File: industry2/__main__.py (hunk 1 above)
Comment (pawelfor) [reply industry2/__main__.py:9]: https://github.com/tornadoweb/tornado/issues/2608#issuecomment-550180288

**Review by Sorebit — APPROVED** [review]
Works on linux

**Pull request merged**